lib/server.js

```diff
@@ -72,8 +70,8 @@ function read(req) {
 function parsePOST(req) {
     const readIt = (req.method === 'PUT') ||
         (req.method === 'POST' && req.headers &&
-            (/^application\/json/i.test(req.headers['content-type'])
-            || !req.headers['content-type']));
+            (/^application\/json/i.test(req.headers['content-type']) ||
+            !req.headers['content-type']));
```

Comment:
It’s nice to give it one more indent for readability

Reply:
Done